tox.ini

```diff
@@ -42,6 +42,10 @@ commands =
         {toxinidir}/runtimes/lightgbm \
         {toxinidir}/runtimes/mlflow \
         {toxinidir}/runtimes/huggingface
+set_env =
```

Comment:
ditto

Reply:
See the `mlserver` env which requires that for some env test. Feels a bit weird to add some comments there.

Reply:
oh I see, we missed this change from [here](https://github.com/SeldonIO/MLServer/blob/b848836872a215cc781075afdbcac345bea49ab6/tox.ini#L24) to be added to `all-runtimes`